### DISCUSSION TITLE
No chia won in 4 meses whats going on

### DISCUSSION THREAD

**Discussion by tkmbe99** [Chia-Network/chia-blockchain#19050] [Q&A]
Since 12.8.2024 I have not won any chia. I have 1162 plots (115 TiB)
Estimated chia earn: 0.84/month

The plots are realy old ones.
I'm using 2.4.4

Any issue?

**Comment by BrandtH22** [comment]
Hey @tkmbe99 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

- Can you share your log file from `~/.chia/mainnet/log/debug.log`?
- Are you pooling? (when pooling users get more information on the status of their farm)
- What OS are you running?

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
